Formula/scotch.rb

```diff
@@ -0,0 +1,63 @@
+class Scotch < Formula
+  desc "Graph and mesh partitioning, clustering, and sparse matrix ordering"
+  homepage "https://gforge.inria.fr/projects/scotch"
+  url "https://gforge.inria.fr/frs/download.php/file/34618/scotch_6.0.4.tar.gz"
+  sha256 "f53f4d71a8345ba15e2dd4e102a35fd83915abf50ea73e1bf6efe1bc2b4220c7"
+
+  option "without-test", "skip build-time tests (not recommended)"
+
+  depends_on "open-mpi"
+  depends_on "xz" => :optional # Provides lzma compression.
+
+  def install
+    ENV.deparallelize if MacOS.version >= :sierra
```

Comment:
Why? Was the bug reported?

Reply:
Not sure. I just found this issue https://github.com/Homebrew/homebrew-science/issues/4382.

It seems the bug hasn't been reported to the [upstream](https://gforge.inria.fr/tracker/?atid=1079&group_id=248&func=browse). So I created an [issue](https://gforge.inria.fr/tracker/index.php?func=detail&aid=21597&group_id=248&atid=1079).

Reply:
FYI, the upstream confirmed that the issue is fixed: https://gforge.inria.fr/tracker/?func=detail&atid=1079&aid=21597&group_id=248